playbooks/runcible.yml

```diff
@@ -36,10 +36,10 @@
       become: true
 
     - name: "Install RVM gpg key"
-      shell: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
+      command: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
 
     - name: "Install RVM"
-      shell: "curl -sSL https://get.rvm.io | bash -s stable --ruby"
+      command: "curl -sSL https://get.rvm.io | bash -s stable --ruby"
```

Comment:
a `command` can't contain a pipe

Reply:
Bleh. Should we just remove this playbook? I'm not sure anyone really uses it. Especially now that all the focus is on removing Pulp 2.

Reply:
Yes, you can remove this playbook.